dscreator/datasets/trajectories/ferrybox.py

```diff
@@ -20,8 +20,8 @@ def dataset_attributes(self, ds: xr.Dataset) -> FerryboxDatasetAttrs:
         return FerryboxDatasetAttrs(
             title="FerryBox on MS Color Fantasy",
             title_no="FerryBox på MS Color Fantasy",
-            summary="Ferry sailing from Oslo, Norway to Kiel, Germany. For more information see: https://www.colorline.no/oslo-kiel/fakta.",
-            summary_no="Ferje fra Oslo, Norge til Kiel, Tyskland. For mer informasjon se: https://www.colorline.no/oslo-kiel/fakta.",
+            summary="The FerryBox system measures temperature, salinity, oxygen, chlorophyll and particle content at a depth of ~5m along the route of MS Color Fantasy Oslo-Kiel. This amounts to about one measurement every ~500 metres. For more information see https://www.niva.no/en/ferrybox",
```

Comment:
where is this text from?

Reply:
More or less from here https://www.niva.no/en/ferrybox